### PR TITLE
Add plugin editor

### PR DESCRIPTION
**Pull request by lavalse**
I've added a separate page about the plugin editor. please help me to check the English. And if something not good, just let me know.

### CODE REVIEW

File: developer-guide/plugin/plugin-editor.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+You can also try our new [Plugin Editor](https://app.reearth.io/plugin-editor) to aid you in developing your own plugin:
+
+<iframe width="100%" height="640" src="https://app.reearth.io/plugin-editor" title="Re:Earth Plugin Editor"></iframe>
+
+## Introduction 
+
+How to make the process of developing plugins simple has always been a problem that our team wants to solve. The main problem is that there is no real-time feedback during the development process, so developers have to repeatedly install and update the plugin. This tool can help us a lot.
+
```
Comment (KaWaite) [inline developer-guide/plugin/plugin-editor.md:13]:
```suggestion
Up until now, the process of developing plugins has had a couple speed bumps that our team has wanted to solve. The main one being that there was no real-time feedback during the development process, so developers have to repeatedly install and update the plugin. Depending on the scope of your plugin, this tool can help a lot.


```

File: developer-guide/plugin/plugin-editor.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Modify your plugin in the code box on the right, and you can see the effect in real time in the left window. This is very helpful for debugging interface and style.
```
Comment (KaWaite) [inline developer-guide/plugin/plugin-editor.md:14]:
```suggestion
Modify your plugin in the code box on the right, and you can see the changes happen in real time in the left window. This is very helpful for updates to the interface and smaller style tweaking.

```

File: developer-guide/plugin/plugin-editor.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Still missing
+
+- Calls to the Re:Earth api are not yet supported
+- Still does not support modifying the yml file to customize the interface with the system (right panel property)
```
Comment (KaWaite) [inline developer-guide/plugin/plugin-editor.md:19]:
```suggestion
- Still does not support modifying the yml file (that is used to display customizable plugin fields in the right panel while editing the project in Re:Earth)

```

File: developer-guide/plugin/plugin-editor.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- Code highlighting is still not supported
```
Comment (KaWaite) [inline developer-guide/plugin/plugin-editor.md:20]: If you mean basic syntax highlighting, that is currently supported (for Typescript). Or do you mean something else by this?

File: developer-guide/plugin/plugin-editor.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## In the feature
+
+We will find ways to improve the Plugin Editor and improve the convenience of the development process. We also need your suggestions and ideas to help us.
```
Comment (KaWaite) [inline developer-guide/plugin/plugin-editor.md:24]:
```suggestion
We will continue to find ways to improve and add to the Plugin Editor to make the plugin development process more streamlined. We also welcome your suggestions and ideas to help us, so please go to Discord or GitHub and start a discussion!
```

**Review by KaWaite — COMMENTED** [review]
A couple changes and one comment!

File: developer-guide/plugin/plugin-editor.md (hunk 4 above)
Comment (lavalse) [reply developer-guide/plugin/plugin-editor.md:20]: I mean for the html highlight ...

**Pull request merged**